### PR TITLE
Issue #79:  Added Swipe functionality and delete calls for Dose Series

### PR DESCRIPTION
**Pull request by jking31cs**
Added the abilty to swipe over dose series and call delete.  This
triggers a delete on that dose series, publishes the delete event,
which is then subscribed by the main list, causing a reload of the
series.  Also did some minor UI cleanup.

### CODE REVIEW

File: app/dose-series-service/dose-series.service.ts [hunk 1]
```diff
@@ -49,6 +49,23 @@ export class DoseSeriesService {
     }
 
     /**
+     * Deletes a DoseSeries from the server.
+     * @param id to lookup.
+     * @returns {Promise<DoseSeries>}.
+     */
+    public delete(
```
Comment (jking31cs) [inline app/dose-series-service/dose-series.service.ts]: This wasn't there before, adding it in.

File: app/dose-series-slider/dose-series-slider.component.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    constructor(
+        private doseSeriesService: DoseSeriesService,
+        private events: Events
+    ) {
+    }
+
+    slidingDelete(slidingItem: ItemSliding) {
+        this.doseSeriesService.delete(this.doseSeries.id).then(
+            (doseSeries: DoseSeries) => {
+                this.events.publish(
```
Comment (jking31cs) [inline app/dose-series-slider/dose-series-slider.component.ts]: publishing an event for the main listing to subscribe to, causes a reload of dose series on the page.

**Comment by jking31cs** [comment]
Screen Shots:

screen shot 1 delete
screen shot 2 delete

File: app/dose-series-slider/dose-series-slider.component.html [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<ion-item-sliding #slidingItem *ngIf="doseSeries">
+    <ion-item>
+        <h2>{{doseSeries.med.name}}</h2>
+        <span>Take {{doseSeries.med.doseAmount}} {{doseSeries.med.doseUnit}}</span><span *ngIf="doseSeries.med.doseInstructions">, {{doseSeries.med.doseInstructions}}</span>
+    </ion-item>
+    <ion-item-options>
+        <button primary (click)="slidingDelete(slidingItem)">
+            <ion-icon name="Delete"></ion-icon>
```
Comment (jking31cs) [inline app/dose-series-slider/dose-series-slider.component.html]: how would I go about changing the color from blue to red for delete?

**Comment by jleopold28** [comment]
I already have some similar code to this.....I am planning on merging this and fixing up the UI more on my PR for issue 79

**Pull request merged**